### PR TITLE
added configuration for creating repose-valve external docker image

### PR DESCRIPTION
**Pull request by deep8103**
- Removed deprecated filter ip-identity in Repose-9 (ip-identity.cfg.xml) and added ip-user filters instead.
- Updated http-connection-pool.cfg.xml according to Repose 9.
- Added argument for creating internal/external/custom filters image using single Dockerfile.
-  Updated system-model.cfg.xml filter configuration. 
- To test on local machine usage-schema and feeds-filters*.war needs to be added locally in Dockerfile.

### CODE REVIEW

File: docker/configuration/external/ip-user.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <user-header name="X-PP-User" quality="0.4"/> 
+    <group-header name="X-PP-Groups" quality="0.4"/>
+    <group name="sample-ipv4-group"> 
+        <cidr-ip>0.0.0.0/0</cidr-ip> 
+    </group>
+    <group name="match-all"> 
+        <cidr-ip>0.0.0.0/0</cidr-ip> 
+        <cidr-ip>0::0/0</cidr-ip> 
+    </group>
+</ip-user>
```
Comment (AJStieren) [inline docker/configuration/external/ip-user.cfg.xml:12]: Nitpick: Add an empty line at the end of the file

File: docker/configuration/internal/ip-user.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ip-user xmlns="http://docs.openrepose.org/repose/ip-user/v1.0">
+    <group name="match-all"> 
+        <cidr-ip>0.0.0.0/0</cidr-ip> 
+        <cidr-ip>0::0/0</cidr-ip> 
+    </group>
+</ip-user>
```
Comment (AJStieren) [inline docker/configuration/internal/ip-user.cfg.xml:7]: Nitpick: Add an empty line at the end of the file

**Review by AJStieren — APPROVED** [review]
Image built, verbose curl showed Repose response (500, filter not initialized), and the log showed the valve being spun up. Altogether, nice.

I have a few minor nitpicks on format style/consistency, but nothing to deny approval.

**Comment by AJStieren** [comment]
Oh, one more thing. Mention that the vpn is required for image build in the README as well as artifactory access. We'll need to be more verbose later with a dev onboarding article.

File: docker/Dockerfile [hunk 3]
```diff
@@ -44,4 +47,4 @@ RUN chmod +x /usr/share/repose/filters/feeds-filters-*.ear
 
 EXPOSE 9090 
 # Start Repose.
-CMD java $JAVA_OPTS -jar /usr/share/repose/repose.jar -c /etc/repose && tail -f /var/log/repose/repose.log
+CMD java $JAVA_OPTS -jar /usr/share/repose/repose.jar -c /etc/repose && tail -f /var/log/repose/repose.log
```
[the remark this answers is not in the document]
Comment (deep8103) [reply docker/Dockerfile]: Done.

File: docker/README.md [hunk 4]
```diff
@@ -26,8 +21,19 @@ APP_VARS=/var/repose
 APP_LOGS=/var/log/repose
 ```
 
-Example of build and run a container with schema version 1.137.0 and feeds_filters_version 1.6.0 with a valid saxon license file
+Run the following command to build an image with custom filters only. 
+```
+$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.7.0 --build-arg saxon_lic=saxon-license.lic -f Dockerfile -t cloudfeeds-repose-custom:9.1.0.2 .
```
[the remark this answers is not in the document]
Comment (deep8103) [reply docker/README.md]: Done.

File: docker/README.md [hunk 5]
```diff
@@ -26,8 +21,19 @@ APP_VARS=/var/repose
...
+Run the following command to build an image with custom filters only. 
+```
+$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.7.0 --build-arg saxon_lic=saxon-license.lic -f Dockerfile -t cloudfeeds-repose-custom:9.1.0.2 . 
+```
+
+Run the following command to build an image with custom + external filters. 
 ```
-$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.6.0 --build-arg saxon_lic=saxon-licese.lic -t repose-valve:9.1.0.2 -f Dockerfile .
-$docker run -itd --name repose-valve -p 9090:9090 repose-valve:9.1.0.2
+$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.7.0 --build-arg saxon_lic=saxon-license.lic --build-arg repose_valve=external -f Dockerfile -t cloudfeeds-repose-external:9.1.0.2 .
```
[the remark this answers is not in the document]
Comment (deep8103) [reply docker/README.md]: Done.

File: docker/README.md [hunk 6]
```diff
@@ -26,8 +21,19 @@ APP_VARS=/var/repose
...
+Run the following command to build an image with custom + external filters. 
 ```
-$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.6.0 --build-arg saxon_lic=saxon-licese.lic -t repose-valve:9.1.0.2 -f Dockerfile .
-$docker run -itd --name repose-valve -p 9090:9090 repose-valve:9.1.0.2
+$docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.7.0 --build-arg saxon_lic=saxon-license.lic --build-arg repose_valve=external -f Dockerfile -t cloudfeeds-repose-external:9.1.0.2 . 
+```
+
+Use the following command to run a cloudfeeds repose-valve external container on port 9090
 ```
+$docker run -itd --name [Conatiner_Name] -p 9090:9090 cloudfeeds-repose-external:9.1.0.2
```
[the remark this answers is not in the document]
Comment (deep8103) [reply docker/README.md:36]: Done.

File: docker/configuration/external/ip-user.cfg.xml (hunk 1 above)
Comment (deep8103) [reply docker/configuration/external/ip-user.cfg.xml]: Done.

File: docker/configuration/internal/ip-user.cfg.xml (hunk 2 above)
Comment (deep8103) [reply docker/configuration/internal/ip-user.cfg.xml]: Done.

**Comment by deep8103** [comment]
Merging, PR comments implemented.

**Pull request merged**